### ISSUE TITLE
FIXMEs in `PipeOpTaskPreproc`

### ISSUE THREAD

**Issue by sebffischer** [mlr-org/mlr3pipelines#767]
there are a few FIXMEs in the class:

* https://github.com/mlr-org/mlr3pipelines/blob/a4933e1b925d6d6c9c4a24794a441367040989e8/R/PipeOpTaskPreproc.R#L225
* https://github.com/mlr-org/mlr3pipelines/blob/a4933e1b925d6d6c9c4a24794a441367040989e8/R/PipeOpTaskPreproc.R#L258
* https://github.com/mlr-org/mlr3pipelines/blob/a4933e1b925d6d6c9c4a24794a441367040989e8/R/PipeOpTaskPreproc.R#L211
* https://github.com/mlr-org/mlr3pipelines/blob/a4933e1b925d6d6c9c4a24794a441367040989e8/R/PipeOpTaskPreproc.R#L225

**Comment by sebffischer** [comment]
@mb706 This is related to https://github.com/mlr-org/mlr3pipelines/issues/864 and Max needs this for his time series package

**Comment by m-muecke** [comment]
@mb706 this probably can be closed I can't find any fixmes in `PipeOpTaskPreproc`

**Comment by advieser** [comment]
True, those were removed with https://github.com/mlr-org/mlr3pipelines/pull/866. So, this can be closed.
